alcs-frontend/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "root": true,
   "ignorePatterns": ["projects/**/*"],
+  "no-octal-escape": 0,
```

Comment:
I would prefer we move all the eslintrc changes to another since it is out of scope of this change